Models/NormalBumpBakingMethods/README.body.md

```diff
@@ -0,0 +1,77 @@
+## Screenshot
+
+![Screenshot from Babylon.js Sandbox](screenshot/screenshot_Large.jpg)
+<br/>_Screenshot from [Babylon.js Sandbox](https://sandbox.babylonjs.com/)._
+
+## Description
+
+This asset is for testing the rendering of tangent-space normal bump maps which are "baked" to specific geometries. 
+
+## Tangent Space
+
+Tangent space normal bump maps are often used in two ways: 
```

Comment:
Can we remove the word "bump" here and throughout?  I believe the proper name is "Tangent space normal map".

The terms "bump" and "bump map" are older and less well defined, often referring to height maps, displacement maps, and the like.  Let's avoid saying "bump" here.